apps/docs/pages/guides/database/extensions/postgis.mdx

```diff
@@ -230,7 +230,7 @@ When you are working on a map-based application where the user scrolls through y
 
 ```sql
 create or replace function restaurants_in_view(min_lat float, min_long float, max_lat float, max_long float)
-returns setof record
+returns table (id public.restaurants%TYPE, name public.restaurants%TYPE, location text, dist_meters float)
```

Comment:
Would we need to add the column name as well before the `%` sign? Also, this particular function does not return distance, so we can delete that one. 

```suggestion
returns table (id public.restaurants.id%TYPE, name public.restaurants.name%TYPE, location text)
```